APIWrapper/Proxy.cs

```diff
@@ -0,0 +1,122 @@
+            string address = $"{__URL__}/{__SCHEDULE_CONTENT_URL__}?m={type}&id={Uri.EscapeUriString(requestContent)}";
+
+            HttpWebRequest request = (HttpWebRequest)WebRequest.Create(address);
+            request.Method = "Get";
+            request.KeepAlive = true;
+            request.ContentType = "application/x-www-form-urlencoded";
+
+            HttpWebResponse response = (HttpWebResponse)request.GetResponse();
+            string myResponse = "";
+            using (System.IO.StreamReader sr = new System.IO.StreamReader(response.GetResponseStream()))
+            {
+                myResponse = sr.ReadToEnd();
+            }
+            
+            return ParseResponse(myResponse);
+        }
+
+        private ScheduleWeekContent ParseResponse(string myResponse)
+        {
+            
+            JObject joResponse = JObject.Parse(myResponse);
+            //check if error occured
+            string report = joResponse["report"].ToString();
+            if (!String.IsNullOrWhiteSpace(report))
+            {
+                throw new ArgumentException(report);
+            }
+            
+            JArray array = (JArray)joResponse["ScheduleContent"];
+            ScheduleWeekContent ret = new ScheduleWeekContent();
+            foreach (var element in array)
+            {
+                int blockNumber = 1;
+                var daySchedule = new ScheduleDayContent();
+                foreach (var blck in element)
+                {
+                    ScheduleHourContent sc = null;
+                    try
+                    {
+                        sc = new ScheduleHourContent();
+                        sc.BlockNumber = blockNumber++;
+                        sc.IsBlocked = Convert.ToBoolean(int.Parse(blck["b"].ToString()));
+
+                        if (!String.IsNullOrWhiteSpace(blck["t"].ToString()) && !String.IsNullOrWhiteSpace(blck["p"].ToString()))
+                        {
+                            sc.LessonType = this.ConvertLessonType(blck["t"].ToString()[0]);
+                            sc.TeacherName = blck["u"].ToString();
+                            sc.RoomName = blck["r"].ToString();
+                            sc.SubjectShortcut = blck["s"].ToString();
+                            sc.SubjectName = blck["k"].ToString();
+                            sc.StudyGroups = blck["g"].ToString().Split(',').Select(x => x.Trim()).ToList();
+                            sc.SubjectType = (eSubjectType)Convert.ToInt32(blck["p"].ToString());
+                            sc.IsEmptyBlock = false;
+                        }
+                        else
+                        {
+                            sc.IsEmptyBlock = true;
+                        }
+                        daySchedule.BlocksInDay.Add(sc);
+                    }
+                    catch (Exception ex)
+                    {
+                        //to do logger
+                        Console.WriteLine(ex.ToString());
```

Comment:
Use NLog. It is already prepared in WebAPI project, so it should be pretty quick to copy configuration...

Reply:
Nlog is not compatible with .net standard :/

Reply:
In that case you should re-throw exception to higher scope. Ignoring exceptions just screams: "no no". You probably are aware of possible cases where you could get exception and have it on top of your mind, but others do not.
Why are you not using some library for automatic object mapping (deserialization)? It would be much more clean and more future proof for changes.

Reply:
One more thing, I do not like that on one line you use 'var' and on other concrete object type. Choose one and stick to it 😃.

Reply:
Support
NLog supports the following platforms:
.NET Framework 3.5, 4, 4.5, 4.6 & 4.7
.NET Framework 4 client profile
Xamarin Android
Xamarin iOs
Windows Phone 8
Silverlight 4 and 5
Mono 4
ASP.NET 4 (NLog.Web package)
ASP.NET Core (NLog.Web.AspNetCore package)
.NET Core (NLog.Extensions.Logging package)
.NET Standard 1.x - NLog 4.5
.NET Standard 2.x - NLog 4.5
UWP - NLog 4.5

Reply:
![nlog](https://user-images.githubusercontent.com/26389312/37334829-4784dd98-26ad-11e8-9628-75a6274b4b60.png)
I cant find it on NuGet :/

Reply:
Add manually to csproj:
```
  <ItemGroup>
    <PackageReference Include="NLog.Web.AspNetCore" Version="4.5.0-rc3" />
    <PackageReference Include="NLog" Version="4.5.0-rc06" />
  </ItemGroup>
```
Read more [here](https://github.com/NLog/NLog.Web/wiki/Getting-started-with-ASP.NET-Core-2#1-add-dependency-in-csproj-manually-or-using-nuget).